spec/support/helpers/state_file_intake_helper.rb

```diff
@@ -75,6 +79,7 @@ def expect_programmatically_associated_help_text
   end
 
   def step_through_initial_authentication(check_a11y: false, contact_preference: :text_message)
+    expect(page).to have_current_path("/en/questions/contact-preference")
     expect(page).to have_text I18n.t("state_file.questions.contact_preference.edit.title")
```

Comment:
why not use the `page_change_check` here?

Reply:
Oh good catch yes I will change that!